mock-core-configs/mock-core-configs.spec

```diff
@@ -18,7 +18,7 @@ BuildArch:  noarch
 Provides: mock-configs
 
 # distribution-gpg-keys contains GPG keys used by mock configs
-Requires:   distribution-gpg-keys >= 1.48
```

Comment:
I don't see this update, yet.  It means that the testing package wouldn't be installable.  Otherwise LGTM

Reply:
Yeah, I raised it to the next version because there are two PRs pending for it:

* https://github.com/xsuchy/distribution-gpg-keys/pull/45
* https://github.com/xsuchy/distribution-gpg-keys/pull/46